lib/models/bundle.js

```diff
@@ -293,20 +298,25 @@ Bundle.create = function (opts, callback) {
       return void callback(readErr);
     }
 
-    // Create and validate.
-    const bundle = new Bundle({
-      path: opts.bundle,
-      code: data.toString(),
-      allowEmpty: opts.allowEmpty
-    });
+    let bundle;
```

Comment:
This chunk looks mostly identical to the one above–maybe we can break it out into a function?

Reply:
Done.